sigma/backends/loki/loki.py

```diff
@@ -227,38 +194,70 @@ class LogQLBackend(TextQueryBackend):
     filter_chars: ClassVar[str] = ""
 
     field_replace_pattern: ClassVar[Pattern] = re.compile("[^a-zA-Z0-9_:]+")
-    field_null_expression: ClassVar[str] = "{field}=``"
 
-    # LogQL does not support wildcards, so we convert them to regular expressions
-    wildcard_multi: ClassVar[
-        str
-    ] = "*"  # Character used as multi-character wildcard (replaced with .*)
-    wildcard_single: ClassVar[
-        str
-    ] = "?"  # Character used as single-character wildcard (replaced with .)
+    negated_line_filter_operator: ClassVar[Dict[str, str]] = {
+        "|=": "!=",
+        "!=": "|=",
+        "|~": "!~",
+        "!~": "|~",
+    }
+
+    currentTemplates: ClassVar[Union[bool, None]] = None

```

Comment:
If we switch to Python 3.10, we can make all these type information more readable like below:
```suggestion
    currentTemplates: ClassVar[bool | None] = None
```

Reply:
Yes, but pySigma and sigma-cli both support Python 3.8, where this syntax won't work.

Reply:
Maybe we can ask them to upgrade.